CHANGELOG.md

```diff
@@ -113,18 +18,33 @@ New types that wrap native types from Apple, Google and Amazon, and we cleaned u
 - `reset` has been removed in favor of `logOut`.
 - `getEntitlements` has been removed in favor of `getOfferings`.
 - `attributionKey` and `Purchases.addAttributionData` have been removed in favor of `set<NetworkID> methods`.
+- `setAllowSharingStoreAccount` has been removed. Configure behavior through the RevenueCat dashboard instead.
 
 ### Renamed APIs
-**Note:** These still might change between `3.0.0-rc.x` and the final release of `3.0.0`
-
 | 2.x | 3.0.0 |
 | :-: | :-: |
 | `PurchaserInfo` | `CustomerInfo` |
+| `Purchases.getPurchaserInfo` | `Purchases.getCustomerInfo` |
+| `Purchases.invalidatePurchaserInfoCache` | `Purchases.invalidateCustomerInfoCache` |
 | `PurchasesTransaction` | `PurchasesStoreTransaction` |
+| `PurchasesTransaction.revenueCatId` | `PurchasesStoreTransaction.transactionIdentifier` |
+| `PurchasesTransaction.productId` | `PurchasesStoreTransaction.productIdentifier` |
 | `PurchasesProduct` | `PurchasesStoreProduct` |
-| `Purchases.RestoreTransactions` | `Purchases.restorePurchases` |
-| `Purchases.GetPaymentDiscount` | `Purchases.getPromotionalOffer` |
```

Comment:
I am surprised about this, but this function is not in the plugin, only the `PurchasesPromotionalOffer` interface

Reply:
Removed the interface in https://github.com/RevenueCat/cordova-plugin-purchases/pull/196 since we don't have the functions to use it